packages/rest-typings/src/v1/omnichannel.ts

```diff
@@ -1175,6 +1175,26 @@ const GETOmnichannelContactSearchSchema = {
 
 export const isGETOmnichannelContactSearchProps = ajv.compile<GETOmnichannelContactSearchProps>(GETOmnichannelContactSearchSchema);
 
+type POSTLivechatAgentStatusProps = { status?: ILivechatAgent['statusLivechat']; agentId?: string };
+
+const POSTLivechatAgentStatusPropsSchema = {
+	type: 'object',
+	properties: {
+		status: {
+			type: 'string',
+			enum: ['available', 'not-available'],
```

Comment:
I think it would be better to rely on the type definition, wdyt?
```suggestion
			enum: Object.values(ILivechatAgentStatus),
```

Reply:
I mean, it won't hurt nobody :trollface: